flows/nodes/scriggo.md

```diff
@@ -0,0 +1,378 @@
+```
+
+### Other Package Types
+
+| Name     | Type/Prototype      | Description                                                    |
+|----------|---------------------|----------------------------------------------------------------|
+| `START`    | [time.Time](https://pkg.go.dev/time#Time) | The scheduled start time for the flow. This time may be far in the past if the flow is backfilling missed runs. |
+| `LAST_RUN` | [time.Time](https://pkg.go.dev/time#Time) | The timestamp of the last run of this flow. |
+| `DURATION` | [time.Duration](https://pkg.go.dev/time#Duration) | The scheduled time duration of a flow run. This repesents the time window the flow should cover for any queries. |
+| `ThisScriptID` | `string` | The current flow ID |
+| `GetClient` | `func() *client.Client` | Returns an active logged in client for use, [client.Client](https://pkg.go.dev/github.com/gravwell/gravwell/v3/client#Client) docs available. |
+| `SetPersistentMap` | `SetPersistentMap(mapName string, key string, value interface{})` | Set a natively typed value into a persistent map that can be retrieved across runs. |
+| `GetPersistentMap` | `GetPersistentMap(mapName string, key string) (value interface{})` | Get a persistent map value from previous runs. |
+| `DeletePersistentMap` | `DeletePersistentMap(mapName, value string)` | Delete a value from the persistent map. |
+| `PersistentMap` | `PersistentMap(mapName string) map[string]interface{}` | Get a complete persistent map. |
+| `Backup` | `Backup(wtr io.Writer, includeSS bool) (err error)` | Create a new backup with a provided `io.Writer`. |
+| `BackupWithConfig` | `BackupWithConfig(wtr io.Writer, cfg types.BackupConfig) (err error)` | Create a new backup with the provided `io.Writer` and [BackupConfig](https://pkg.go.dev/github.com/gravwell/gravwell/v3/client/types#BackupConfig). |
+| `GetMacro` | ` GetMacro(key string) (string, error)` | Retrieve a system macro. |
+| `Systems` | `Systems() (map[string]types.SysInfo` | Get a map of [SysInfo](https://pkg.go.dev/github.com/gravwell/gravwell/v3/client/types#SysInfo) structures using the system name as the map key. |
+| `Ingesters` | `Ingesters() (map[string]types.IngestStats, error)` | Get a map of [IngesterStats](https://pkg.go.dev/github.com/gravwell/gravwell/v3/client/types#IngesterStats) keyed by terminating indexer UUID. |
+| `Indexers` | `Indexers() (map[string]types.IdxStats, error)` | Get a map of [IdxStats](https://pkg.go.dev/github.com/gravwell/gravwell/v3/client/types#IdxStats) keyed by indexer UUID.   This API call is expensive and returns the complete shard state systemwide.  |
+| `IndexerStates` | `IndexerStates() (map[string]string, error)` | Get a map of indexer health states keyed by indexer UUID.|
+| `AddSelfTargetedNotification` | `AddSelfTargetedNotification(id uint32, msg, link string, expires time.Time) error` | Create a Gravwell notification. |
+| `GetResource` | `GetResource(key string) ([]byte, error)` | Get a gravwell resource. |
```

Comment:
Gravwell is capitalized in other descriptions.

Reply:
fixed